shell/browser/api/electron_api_system_preferences.h

```diff
@@ -76,17 +76,17 @@ class SystemPreferences
   void PostNotification(const std::string& name,
                         base::DictionaryValue user_info,
                         gin::Arguments* args);
-  int SubscribeNotification(const std::string& name,
+  int SubscribeNotification(v8::Local<v8::Value> maybe_name,
```

Comment:
`absl::optional` perhaps?

Reply:
@nornagon `absl::optional` doesn't handle invalid values it handles missing values - so passing `null` with an optional string here would throw the same conversion error.